autorest/sender_test.go

```diff
@@ -144,7 +144,7 @@ func TestSend(t *testing.T) {
 func TestAfterDelayWaits(t *testing.T) {
 	client := mocks.NewSender()
 
-	d := 5 * time.Millisecond
```

Comment:
Yes, milliseconds with above change was giving zero seconds delay so increased it.
```Go
backoff := 5* time.Millisecond
fmt.Println(backoff.Seconds())
fmt.Println(time.Duration(backoff.Seconds()))

//Output:
0.005
0

```